Adds the ability to select which language faker will generate the data

rleger opened the pull request:
It would be nice to be able to select which language Faker will generate the factory data from.

I thought of using the app.locale var but what if the language is not available in faker... might be cleaner to differentiate the two.

So instead I thought you could add a dedicated config option in the database file for instance. That way you can swap it easily.

Sorry if this is not very clean, just trying to contribute in my own little way with a solution to something that's bothering me.

Thank you.

config/database.php

```
    /*
    |--------------------------------------------------------------------------
    | Faker locale
    |--------------------------------------------------------------------------
    |
    | Language in which faker will generate dummy data
    |
    */

    'faker_locale' => 'fr_FR', 
```

rleger committed:
Adds the ability to select which language faker will generate its dummy data in

rleger committed:
missing space

rleger committed:
removing extra space...

GrahamCampbell commented on src/Illuminate/Database/DatabaseServiceProvider.php:
```diff
@@ -60,7 +60,7 @@ public function register()
     {
         $this->app->singleton(FakerGenerator::class, function () {
-            return FakerFactory::create();
+        	return FakerFactory::create(config('database.faker_locale', 'en_US'));
```
please don't change the indentation

GrahamCampbell replied:
you can't use the config helper function from a service provider not dependent on the foundation helpers

rleger committed:
Set faker language

rleger committed:
Merge branch '5.1' of https://github.com/rleger/framework into 5.1

rleger commented:
Of course.. sorry, removed the helper function call and replaced it with $app['config']. Sorry about the indentation, I got an error from StyleCI. Thanks

taylorotwell commented:
I really prefer to not have another config option for this.

rleger commented:
ok, how about a "service" that would try to initialize faker with the language set in app.locale and default to en_US ? Right now I'm overriding the binding in my own AppServiceProvider which is fine but a bit hacky.

taylorotwell commented:
I don’t think that’s really hacky at all. That’s the purpose of the IoC container.

On Mon, Oct 5, 2015 at 1:48 PM, Romain Léger <notifications@github.com>
wrote:

> ok, how about a "service" that would try to initialize faker with the language set in app.locale and default to en_US ? Right now I'm overriding the binding in my own AppServiceProvider which is fine but a bit hacky. 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/laravel/framework/pull/10490#issuecomment-145629270

rleger commented:
alright, fine by me. Thanks for your reply
